client/components/login/__snapshots__/login.test.js.snap

```diff
@@ -53,14 +53,10 @@ exports[`<Login /> rendering should render correctly with social links 1`] = `
         >
           <React.Fragment>
             <label
-              className="owisp-login-label owisp-login-label-email"
+              className="owisp-login-label owisp-login-label-text owisp-login-label-email"
               htmlFor="owisp-login-email"
             >
-              <div
```

Comment:
Almost there, I meant we should simply substitute label with div for all the occurrences, eg:

```html
<label for="input-name"><div>email</div><input /></label> 
```

would become

```html
<div><label for="input-name">email</label><input /></div>
```

Is it clearer now?

Reply:
Hi @nemesisdesign, thank you for that! I'll double-check why the build is failing along with these changes and resubmit by tonight.

One question about this structure:
```html
<div><label for="input-name">email</label><input id="input-name" /></div>
```

Originally the structure was something like this, eg. for `registration.js`:
```html
<label class="registration-label" for="input-name"><div class="registration-label-text">email</div><input id="input-name" /></label>
```
where the inner div had a class to style the label. 

Just to clarify, do you want to retain the same className when making the inner div the outer div? i.e.
```html
<div class="registration-label-text"><label class="registration-label" for="input-name">email</label><input id="input-name" /></div>
```

Or should the div have no class? Currently, I'd resorted to adding the `.registration-label-text` class to the label so it would maintain the same styling.

Reply:
What you propose seems fine, just ensure the result looks identical to the current look, thanks.